src/packages/@ncigdc/tableModels/utils.tsx

```diff
@@ -83,36 +96,55 @@ export const createDataCategoryColumns = ({
           >
             {_.sumBy(
               hits.edges,
-              x =>
+              (x: any) =>
                 findDataCategory(category.abbr, x.node.summary.data_categories)[
                   countKey
-                ],
+                ]
             ).toLocaleString()}
           </Link>
         </TdNum>
       ),
     })),
   ];
 };
-
-type TCreateSelectColumn = ({
-  idField: string,
-  headerRowSpan?: number,
-}) => {};
-export const createSelectColumn: TCreateSelectColumn = ({
+export interface IColumnProps<NoTH> {
+  name: string;
+  id: string;
+  sortable: boolean;
+  downloadable: boolean;
+  hidden: boolean;
+  field?: string;
+  subHeading?: boolean;
+  subHeadingIds?: NoTH extends true? string[] : undefined;
+  parent?: string;
+  th: (props: any) => JSX.Element;
```

Comment:
There should be an example of th props in src/packages/@ncigdc/modern_components/CancerDistributionTable/CancerDistributionTable.model.tsx

`th: (props: { style?: React.CSSProperties }) => JSX.Element | string;`